airqo-platform/AirQo-api
# [device-status] refactor device-status to use new recent feeds url

#### Summary of Changes (What does this PR do?)
- Change recent device status URL
- Remove unused code
- Flatten directories
- Add ReadMe.md file
- General refactoring
#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch this branch locally.
* Change directory to the `device-status` micro-service directory.
* Follow the steps in the `README.md` file to run the script.
* The script should run successfully.

#### What are the relevant tickets?
- [PLAT-569](https://airqoteam.atlassian.net/browse/PLAT-569)

#### Screenshots (optional)
